### ISSUE TITLE
NullPointerException

### ISSUE THREAD

**Issue by iapilgrim** [magro/msm-sample-webapp#2]
I follow the README.
mvn package
$ ./runtime/tomcat1/bin/catalina.sh run
http://localhost:8081/
Goto StatefulPage
got the error
ERROR - RequestLogger              - Exception while determining the size of the session in the request logger: null
java.lang.NullPointerException
	at java.util.Locale.writeObject(Locale.java:2043)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at java.io.ObjectStreamClass.invokeWriteObject(ObjectStreamClass.java:988)
	at java.io.ObjectOutputStream.writeSerialData(ObjectOutputStream.java:1495)
	at java.io.ObjectOutputStream.writeOrdinaryObject(ObjectOutputStream.java:1431)
	at java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1177)
	at java.io.ObjectOutputStream.defaultWriteFields(ObjectOutputStream.java:1547)
	at java.io.ObjectOutputStream.writeSerialData(ObjectOutputStream.java:1508)
	at java.io.ObjectOutputStream.writeOrdinaryObject(ObjectOutputStream.java:1431)
	at java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1177)
	at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:347)
	at org.apache.wicket.util.lang.Objects$SerializingObjectSizeOfStrategy.sizeOf(Objects.java:89)
	at org.apache.wicket.util.lang.Objects.sizeof(Objects.java:1194)
	at org.apache.wicket.Session.getSizeInBytes(Session.java:820)
	at org.apache.wicket.protocol.http.RequestLogger.requestTime(RequestLogger.java:228)
	at org.apache.wicket.RequestCycle.detach(RequestCycle.java:1163)
	at org.apache.wicket.RequestCycle.steps(RequestCycle.java:1454)
	at org.apache.wicket.RequestCycle.request(RequestCycle.java:545)
	at org.apache.wicket.protocol.http.WicketFilter.doGet(WicketFilter.java:479)
	at org.apache.wicket.protocol.http.WicketFilter.doFilter(WicketFilter.java:312)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at de.javakaffee.web.msm.RequestTrackingContextValve.invoke(RequestTrackingContextValve.java:99)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at de.javakaffee.web.msm.RequestTrackingHostValve.invoke(RequestTrackingHostValve.java:151)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:298)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:859)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:588)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:489)
	at java.lang.Thread.run(Thread.java:744)
INFO  - RequestLogger              - time=254,event=BookmarkablePage[de.javakaffee.msm.bench.HomePage()],response=PageRequest[org.apache.wicket.markup.html.pages.ExceptionErrorPage(10)],sessionid=75B8D625FEE8317FEE8814BE167D301C,sessionsize=-1,sessionstart=Fri May 16 15:28:01 ICT 2014,requests=2,totaltime=254,activerequests=0,maxmem=1908M,total=128M,used=10M

**Comment by csarmiento** [comment]
Same issue here

```
ERROR - RequestLogger              - Exception while determining the size of the session in the request logger: null
java.lang.NullPointerException
	at java.util.Locale.writeObject(Locale.java:2043)
	at sun.reflect.GeneratedMethodAccessor19.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at java.io.ObjectStreamClass.invokeWriteObject(ObjectStreamClass.java:988)
	at java.io.ObjectOutputStream.writeSerialData(ObjectOutputStream.java:1495)
	at java.io.ObjectOutputStream.writeOrdinaryObject(ObjectOutputStream.java:1431)
	at java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1177)
	at java.io.ObjectOutputStream.defaultWriteFields(ObjectOutputStream.java:1547)
	at java.io.ObjectOutputStream.writeSerialData(ObjectOutputStream.java:1508)
	at java.io.ObjectOutputStream.writeOrdinaryObject(ObjectOutputStream.java:1431)
	at java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1177)
	at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:347)
	at org.apache.wicket.util.lang.Objects$SerializingObjectSizeOfStrategy.sizeOf(Objects.java:89)
	at org.apache.wicket.util.lang.Objects.sizeof(Objects.java:1194)
	at org.apache.wicket.Session.getSizeInBytes(Session.java:820)
	at org.apache.wicket.protocol.http.RequestLogger.requestTime(RequestLogger.java:228)
	at org.apache.wicket.RequestCycle.detach(RequestCycle.java:1163)
	at org.apache.wicket.RequestCycle.steps(RequestCycle.java:1454)
	at org.apache.wicket.RequestCycle.request(RequestCycle.java:545)
	at org.apache.wicket.protocol.http.WicketFilter.doGet(WicketFilter.java:479)
	at org.apache.wicket.protocol.http.WicketFilter.doFilter(WicketFilter.java:312)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at de.javakaffee.web.msm.RequestTrackingContextValve.invoke(RequestTrackingContextValve.java:99)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at de.javakaffee.web.msm.RequestTrackingHostValve.invoke(RequestTrackingHostValve.java:151)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:298)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:859)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:588)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:489)
	at java.lang.Thread.run(Thread.java:745)
INFO  - RequestLogger              - time=7,event=null,response=PageRequest[org.apache.wicket.markup.html.pages.ExceptionErrorPage(2)],sessionid=087F72512ED3075D3604C2B10AB44D1C,sessionsize=-1,sessionstart=Thu Feb 19 10:51:56 COT 2015,requests=4,totaltime=25,activerequests=0,maxmem=3817M,total=257M,used=67M
```

**Comment by csarmiento** [comment]
@magro any ideas? Thanks.

**Comment by csarmiento** [comment]
This issue can be solved just updating the msm & kyro libraries to 1.8.3

**Comment by magro** [comment]
Resolved by PR #3
